.env_SAMPLE

```diff
@@ -25,7 +25,7 @@ export DJANGO_HTTP_PORT=8000
 #export CFGOV_PROD_DB_LOCATION=<some_database_dump_url>
 
 # Password given to 'admin' user created by initial-data.sh script.
-#export WAGTAIL_ADMIN_PW=<wagtail_admin_user_password>
+export WAGTAIL_ADMIN_PW=admin
```

Comment:
I changed this to have a default `admin` value as it makes it easier to get started from scratch. With a blank admin password it's not possible to login to the Wagtail admin, so we might as well default this to something.